operator/apis/operator/v1alpha1/oapserverconfig_types.go

```diff
@@ -0,0 +1,76 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OAPServerConfigSpec defines the desired state of OAPServerConfig
+type OAPServerConfigSpec struct {
+	// Version of OAP.
+	//+kubebuilder:validation:Required
+	Version string `json:"version,omitempty"`
+	// StaticConfig holds the OAP server static configuration.
+	// +kubebuilder:validation:Optional
+	StaticConfig []corev1.EnvVar `json:"staticConfig,omitempty"`
```

Comment:
`StaticConfig` is more than env vars. Most of the files lay in [resources](https://github.com/apache/skywalking/tree/master/oap-server/server-starter/src/main/resources) are static ones. A potential solution is to put configuration files from the same subdirectory,  or a single in the root directory, to an independent CR.

Reply:
Indeed, If we want to update the static file, we need to create a CR corresponding to a static file and mount it through Configmap.